Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -331,6 +370,21 @@ GDCMImageIO::Read(void * pointer)
     // WARNING: sizeof(Real World Value) != sizeof(Stored Pixel)
     len = len * outputpt.GetPixelSize() / pixeltype.GetPixelSize();
   }
+  // Y'CbCr to RGB
+  //
+  // TODO: clarify with GDCM YBR_FULL_422 for lossy Jpeg,
```

Comment:
> TODO: clarify with GDCM YBR_FULL_422 for lossy Jpeg,

Something that may be unusual coming from DCMTK, is that GDCM behavior acts as `dcmdjpeg +cn` (which is not the default for DCMTK, but is the default for GDCM).

So let's take an input such as `US-GE-4AICL142.dcm` (JPEG Baseline (Process 1) + YBR_FULL_422). It will be decompressed as Explicit VR Little Endian + YBR_FULL. I agree this is non-standard for viz-people, but keep in mind GDCM was meant for ITK (quantitative analysis).

Reply:
@malaterre 
It happens usually with lossy Jpeg files IMHO, they are almost always YBR_FULL_422 and "greenish blue", and RLE YBR_FULL too. But how process them in GDCM is your decision, we can adjust accordingly. There is an _InputYBRtoRGB_ option, if somebody would like extract [luma](https://drive.google.com/file/d/1zY1QX0CqJwezpQCYy2ZPhDg24Gm-x6Fh/view?usp=sharing).
This PR is WIP, but in fact ready IMHO to use, the places in question (always about lossy Jpeg) are clearly marked as such, i've added functions for Y'CbCr, they could be replaces later with equivalent somewhere from. If you would kindly take a look were great. Thank you very much.

Reply:
Here is US-GE-4AICL142.dcm
![20191007-161719](https://user-images.githubusercontent.com/1494580/66319793-00030780-e91e-11e9-8ad4-99776eb749c0.png)

It has, BTW, a kind of color bar for better view. Bad idea to save shades of gray in lossy Jpeg, but in fact, that trick with [luma](https://drive.google.com/file/d/1zY1QX0CqJwezpQCYy2ZPhDg24Gm-x6Fh/view?usp=sharing) works, very very seldom used, i think.

Reply:
Weasis
![Screenshot at 2019-10-07 16-28-16](https://user-images.githubusercontent.com/1494580/66320664-8b30cd00-e91f-11e9-893e-7aab58185e5e.png)

Reply:
ITK-Snap
![snap](https://user-images.githubusercontent.com/1494580/66356775-93632980-e96c-11e9-8609-1c712efe9511.jpg)